meson.build

```diff
@@ -40,10 +41,16 @@ if ['libidn2', 'auto'].contains(enable_runtime)
 endif
 
 if ['libicu', 'auto'].contains(enable_runtime)
+  libicu_win_dep = cc.find_library('icu', required: false)
   libicu_dep = dependency('icu-uc', 'ICU',
                           components: 'uc',
                           required : false)
-  if libicu_dep.found()
+  if libicu_win_dep.found()
+    libicu_dep = libicu_win_dep
+    if enable_runtime == 'auto'
+      enable_runtime = 'libicu'
+    endif
+  elif libicu_dep.found()
```

Comment:
Should there be an option here to force the use of external icu?

Reply:
I'd say yes, since otherwise it sounds unpredictable to assume the resulting build will work. e.g you provide static ICU libraries in your build environment, upgrade your libpsl clone, and suddenly it starts linking to a different DLL shipped with only certain versions of Windows, so application distribution to older Windows systems breaks.

(Why can't Windows just ship actual forreal ICU...)